tabpy-server/server_tests/test_evaluation_plane_handler.py

```diff
@@ -25,8 +25,11 @@ def setUpClass(cls):
         # create password file
         cls.pwd_file = tempfile.NamedTemporaryFile(
             mode='w+t', prefix=prefix, suffix='.txt', delete=False)
-        cls.pwd_file.write('username {}'.format(
-            hashlib.sha3_256('password'.encode('utf-8')).hexdigest()))
+        username = 'username'
+        password = 'password'
+        cls.pwd_file.write('{} {}\n'.format(
+            username,
+            hash_password(username, 'password')))

```

Comment:
Guess it doesn't matter a lot here but you probably mean to reference the local variable rather than literal string in this expression

Reply:
TY! fixed.